src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("the excavator started it`s work");
```

Comment:
The message should be more informative and indicate that it is an Excavator that is working. Consider rephrasing to something like 'Excavator started working.'